sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -873,54 +891,9 @@ export class MessageReceiver extends LinkEntity {
       }
 
       // We should attempt to reopen only when the receiver(sdk) did not initiate the close
```

Comment:
This comment is now obsolete. It had its place when we used `receiver.isItSelfClosed()`

```suggestion
```

Reply:
Updated the comment!